notebooks/Process_TMPRSS2.ipynb

```diff
@@ -561,20 +561,177 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>88</th>\n",
+       "      <td>90666145</td>\n",
+       "      <td>0.011</td>\n",
+       "      <td>Active</td>\n",
+       "      <td>CNC(=O)NCCC1CCN(CC1)C(=O)[C@H](CC2=CC(=CC=C2)C...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>89</th>\n",
+       "      <td>90666146</td>\n",
+       "      <td>0.003</td>\n",
+       "      <td>Active</td>\n",
+       "      <td>COC1=CC(=C(C=C1)C2=CC(=CC=C2)S(=O)(=O)N[C@@H](...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>90</th>\n",
+       "      <td>90666147</td>\n",
+       "      <td>0.005</td>\n",
+       "      <td>Active</td>\n",
+       "      <td>C1CCC(CC1)NC(=O)NC2CCN(CC2)C(=O)[C@H](CC3=CC(=...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>91</th>\n",
+       "      <td>0</td>\n",
+       "      <td>0.044</td>\n",
+       "      <td>Active</td>\n",
+       "      <td>N=C(C1=CC=C(C=C1)CNC([C@@H]2CCCN2C([C@@H](CCC3...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>92 rows × 4 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         cid  acvalue activity  \\\n",
+       "0   46899577    0.019   Active   \n",
+       "1   56677005    0.019   Active   \n",
+       "2   49835415    0.020   Active   \n",
+       "3   49835557    0.021   Active   \n",
+       "4          0    0.044   Active   \n",
+       "..       ...      ...      ...   \n",
+       "87  90666144    0.013   Active   \n",
+       "88  90666145    0.011   Active   \n",
+       "89  90666146    0.003   Active   \n",
+       "90  90666147    0.005   Active   \n",
+       "91         0    0.044   Active   \n",
+       "\n",
+       "                                               smiles  \n",
+       "0   C1C[C@H](N(C1)C(=O)[C@@H](CCCN=C(N)N)NS(=O)(=O...  \n",
+       "1   CC(C)(C)OC(=O)C[C@H](C(=O)N1CCC[C@H]1C(=O)NCC2...  \n",
+       "2   C1C[C@H](N(C1)C(=O)[C@@H](CCCCN)NS(=O)(=O)CC2=...  \n",
+       "3   C1C[C@H](N(C1)C(=O)[C@@H](CCCCN=C(N)N)NS(=O)(=...  \n",
+       "4   N=C(C1=CC=C(C=C1)CNC([C@@H]2CCCN2C([C@H](CCC3=...  \n",
+       "..                                                ...  \n",
+       "87  CNC(=O)NCCC1CCN(CC1)C(=O)[C@H](CC2=CC(=CC=C2)C...  \n",
+       "88  CNC(=O)NCCC1CCN(CC1)C(=O)[C@H](CC2=CC(=CC=C2)C...  \n",
+       "89  COC1=CC(=C(C=C1)C2=CC(=CC=C2)S(=O)(=O)N[C@@H](...  \n",
+       "90  C1CCC(CC1)NC(=O)NC2CCN(CC2)C(=O)[C@H](CC3=CC(=...  \n",
+       "91  N=C(C1=CC=C(C=C1)CNC([C@@H]2CCCN2C([C@@H](CCC3...  \n",
+       "\n",
+       "[92 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "TMPRSS2"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "for i, row in TMPRSS2.iterrows():\n",
+    "    "
```

Comment:
Looks like you didn't finish whatever you were doing here...

Reply:
Could you add the TMPRSS2 fingerprint dumps here, since you know exactly how to do them?

Reply:
added these (Y)